page-uploader/backend/folder-processor.js

```diff
@@ -1,6 +1,6 @@
 const fs = require('fs');
 const path = require('path');
-const bucketUpload = require('./s3-upload.js');
+let bucketUpload = require('./s3-upload.js');
```

Comment:
I can't spot where `bucketUpload` is ever redefined. Can this be `const` instead of `let`?